File: Core/Data/Team.cs
namespace COA.Data {
  using System;
  using System.Drawing;

  public class Team : IComparable<Team> {
    public string name;
    public Color color;
    public Font font;
    [NonSerialized]
    public int score;

    public int CompareTo(Team other) {
      if(score > other.score) {
        return -1;
      } else if(score < other.score) {
        return 1;
      }
      return 0;
    }
  }
}

File: cuddly-octo-adventure/src/data/team.rs
#[derive(Serialize, Deserialize)]
pub struct Team {
    pub name: String,
    // color: color
    // font: font
    #[serde(skip_serializing, skip_deserializing)]
    pub score: u32,
}

impl Default for Team {
    fn default() -> Team {
        Team {
            name: "Octocats".to_owned(),
            score: 0,
        }
    }
}
